src/app/pages/e-commerce/e-commerce.component.html

```diff
@@ -1,3 +1,19 @@
+<div class="row">
+  <div class="col-12 col-md-6">
+    <ngx-cta-card [title]="'Need a custom solution?'"
+                  [type]="'pantone'"
+                  [link]="'https://www.akveo.com/contacts'"
+                  [linkTitle]="'Contact us'">
+    </ngx-cta-card>
+  </div>
+  <div class="col-12 col-md-6">
+    <ngx-cta-card [title]="'Why ngx-admin works for you?'"
+                  [type]="'briefcase'"
+                  [link]="'https://akveo.github.io/ngx-admin/'"
```

Comment:
too for this link should be `https://akveo.github.io/ngx-admin?utm_source=ngx_admin_demo&utm_medium=contact_card`